src/gameplay/elo/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface RatingBracket {
+  name: string;
+  min: number;
+  max: number;
```

Comment:
Please comment if it's inclusive or not. I think it makes sense for both to be inclusive. 

If you really want, you can name them `minInclusive` and `maxInclusive`.